src/package.lisp

```diff
@@ -87,7 +87,9 @@
 
    #:eassert
 
-   #:prdebug))
+   #:prdebug
```

Comment:
@markhdavid In the future, please examine, critique, and use the `emotiq:note` facility in `emotiq/logging` ASDF system for general messaging streams.